bin/steps/mercurial

```diff
@@ -4,6 +4,6 @@
 if [[ -f "requirements.txt" ]]; then
 	if (grep -Fiq "hg+" requirements.txt) then
 		/app/.heroku/python/bin/pip install  mercurial | cleanup | indent
+		mcount "steps.mercurial"
```

Comment:
You are using `steps.vendor.foo` in other cases, but `steps.mercurial` here. Is that intentional?